app/views/components/_important-metadata.html.erb

```diff
@@ -1,10 +1,16 @@
 <%
+  title ||= false
```

Comment:
It might be clearer to assign this with `title = local_assigns[:title]`, it's slightly misleading to default to a boolean value for a string. Using the value from `local_assigns` will mean you have a string or `nil` and this works fine with the conditional below.